src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ CHANGELOG
    tagged with `workflow.workflow`, and those with type=state_machine with
    `workflow.state_machine`
  * Add `rate_limiter` configuration option to `messenger.transport` to allow rate limited transports using the RateLimiter component
+ * Removed `@internal` tag from secret vaults to allow them to be used directly outside the framework bundle and custom vaults to be added.
```

Comment:
According to the [naming conventions](https://symfony.com/doc/current/contributing/code/conventions.html#writing-a-changelog-entry):
> Messages should follow the [commit message conventions](https://symfony.com/doc/current/contributing/code/pull_requests.html#commit-messages): should be short, capitalize the line, do not end with a period, use an imperative verb to start the line;

So it should be: "Remove [...]".

Reply:
Damn it, @nicolas-grekas was faster than me! 😄

Reply:
thanks anyway 😄